### DISCUSSION TITLE
Täglicher Gesamtertrag wird sporadisch zurückgesetzt

### DISCUSSION THREAD

**Discussion by danka1412** [tbnobody/OpenDTU#675] [Q&A]
Hey zusammen, 

da ich leider noch nicht verstanden habe weshalb es auftritt, mal die Frage an euch Experten:

Bin seit gestern von der DTU Pro weg, da es ja schlichtweg unmöglich ist, Daten vom WR sauber nach HA zu schieben. Also nutze ich nun openDTU und schiebe die Daten via MQTT nach HA. Dabei ist mir aufgefallen, dass der tägliche Gesamtertrag sporadisch auf Null gesetzt wird und der Sensor dann wieder hoch zählt - in der WebApp selbiges.

Hab vorhin auch mal testweise Ahoy geflasht, auch da selbiges Problem, wenn auch etwas konstanter (siehe Plot 11-13 Uhr). Woran könnte es liegen, dass der tägliche Gesamtertrag so sporadisch zurückgesetzt wird? 

![image](https://user-images.githubusercontent.com/117849141/224048933-dcac0406-b4ea-45a3-919a-a6c08c425e94.png)
![image](https://user-images.githubusercontent.com/117849141/224049165-d24aa433-41a8-4b0a-b255-3d6eed52ce36.png)

**Comment by tbnobody** [comment]
Der Inverter startet aufgrund zu wenig Sonnenlicht neu.

**Reply by danka1412** [reply]
Es ist schon sehr diffus heute, dennoch liefern 4 Module durchweg >30V. Sollte dem Kollegen eig reichen. Ich beobachte das aber mal bei besserem Wetter. Danke natürlich erst einmal für die schnelle Antwort.

**Comment by helgeerbe** [comment]
Und wenn du einen echten Tageszähler haben möchtest, dann lege in Home Assistant einen Helferzähler an, der den Gesamtertrag vom Wechselrichter als Eingabe nimmt. 

Der Zähler überlebt dann auch Neustarts z. B. durch Firmware-Updates.

**Comment by ghost** [comment]
Scheint das gleiche Problem wie #630 zu sein.
